dhcpv4/ztpv4/ztp.go

```diff
@@ -15,62 +17,107 @@ type VendorData struct {
 
 var errVendorOptionMalformed = errors.New("malformed vendor option")
 
-// ParseVendorData will try to parse dhcp4 options looking for more
-// specific vendor data (like model, serial number, etc).
-func ParseVendorData(packet *dhcpv4.DHCPv4) (*VendorData, error) {
-	vc := packet.ClassIdentifier()
-	if len(vc) == 0 {
-		return nil, errors.New("vendor options not found")
-	}
+func parseVIVC(packet *dhcpv4.DHCPv4) (*VendorData, error) {
```

Comment:
NIT: Can you move `parseVIVC` below `parseClassIdentifier`?

Github diff will look cleaner and would be nice to have them sorted as option number as well.